.github/workflows/publish_sub_package.yml

```diff
@@ -4,14 +4,18 @@ on:
   push:
     branches:
       - main
+    pull_request:
+      branches: [main]
```

Comment:
Seems like the `publish_sub_packages` workflow has not been successfully triggered. Maybe change to
branches: [main, ipex-llm-llm-gpu]

and have a try again

Reply:
Already added